### PR TITLE
Update foreman-docker to 3.0.0 (DEB)

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.12
* [ ] 1.11
* [ ] 1.10

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by mmoll** [comment]
```
11:56:57 NoMethodError: undefined method `parameter_filter' for #<Foreman::Plugin:0x00000004ce6310>
11:56:57 /usr/share/foreman/vendor/ruby/2.0.0/gems/foreman_docker-3.0.0/lib/foreman_docker/engine.rb:106:in `block (2 levels) in <class:Engine>'
```
What's going on there? :)

**Comment by domcleal** [comment]
That suggests it isn't being tested against the current develop branch, looks like it was a build from the 28th.  release_nightly_test_deb is failing, it's probably due to that (I'll take a look).

**Comment by mmoll** [comment]
[test]

File: plugins/ruby-foreman-docker/debian/gem.list [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_docker-2.1.1.gem"
+GEMS="https://rubygems.org/downloads/foreman_docker-3.0.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/docker-api-1.28.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/wicked-1.1.0.gem"
```
Comment (mmoll) [inline plugins/ruby-foreman-docker/debian/gem.list]: Could these get updated, too? Maybe also in the RPM branch...

Comment (mmoll) [inline plugins/ruby-foreman-docker/debian/gem.list]: @dLobatog ping?

**Comment by mmoll** [comment]
also, the foreman-* versions in the control file should get bumped to `>= 1.13.0~rc1`

**Comment by mmoll** [comment]
cherry-picked as acbc483 in to deb/devleop, gracias @dLobatog!

**Pull request closed without merging**